Evaluator/Social/signal_evaluator/config/TelegramChannelSignalEvaluator_schema.json

```diff
@@ -1,19 +1,31 @@
 {
   "type": "object",
-  "title": "TelegramSignalEvaluator",
-  "additionalProperties" : {
-        "type" : "object",
-        "required" : [
-            "signal_pattern",
-            "signal_market"
+  "title": "TelegramChannelSignalEvaluator",
+  "additionalProperties": {
+    "type": "object",
+    "required": [
+      "signal_pattern",
+      "signal_market"
+    ],
+    "properties": {
+      "signal_pattern": {
+        "type": "object",
+        "required": [
+          "MARKET_BUY",
+          "MARKET_SELL"
         ],
-        "properties" : {
-            "signal_pattern" : {
-                "type" : "string"
-            },
-            "signal_market" : {
-                "type" : "string"
-            }
+        "properties": {
+          "MARKET_BUY": {
+            "type": "string"
+          },
+          "MARKET_SELL": {
+            "type": "string"
```

Comment:
should we add placeholders or descriptions ?

Reply:
We can yes

Reply:
I added titles but I don't think adding placeholders is relevant.